src/node.js

```diff
@@ -118,51 +117,106 @@ module.exports = class Node {
     }
   }
 
-  startDaemon (done) {
-    parseConfig(this.path, (err, conf) => {
+  startDaemon (flags, done) {
```

Comment:
callbacks at the top level of the func should be `callback` just to keep being consistent with the rest of js ecosystem (I do notice that applies to other parts of this module, it just means it was never refactored)